docs/user-guide/02-conventional-features.md

```diff
@@ -17,10 +17,6 @@ Types
 
 Slang supports conventional shading language types including scalars, vectors, matrices, arrays, structures, enumerations, and resources.
 
-> #### Note ####
```

Comment:
We can add a paragraph to mention that Slang has limited support for pointers when targeting platforms with native pointer support.

Reply:
ack